### DISCUSSION TITLE
컴퓨터로 예약 시도 하면서 스마트폰 사용하여 예약 가능한가요?

### DISCUSSION THREAD

**Discussion by ghost** [changdoc/naver-vaccine-macro#104] [Q&A]
개발자님 안녕하세요. 

예약 시도 하는 동안 해당 계정으로 다른 탭, 모바일 등 예약 신청 페이지 접근시 차단 당할 수 있습니다.

라는 말씀은 컴퓨터로 자동예약 돌리는 중에 
스마트폰으로 다른 병원 예약 시도하여 신청페이지까지 접근했을 때에 차단당한다는 뜻으로 이해해도 괜찮을까요?

**Comment by OneTop4458** [comment]
네 맞습니다.
